packages/near-fast-auth-signer/src/components/Login/Login.tsx

```diff
@@ -54,7 +54,7 @@ function Login() {
     data: { email: string }
   ) => {
     const newParams = new URLSearchParams(currentSearchParams);
-    newParams.set('email', encodeURIComponent(data.email));
```

Comment:
I made this fix because I noticed that if we insert email like:
test+1@gmail.com, previous implementation actually returns `test @gmail.com`. with `encodeURIComponent` it actually returns `test+1@gmail.com`. When you revert this change, is there a reason why you have reverted this?

Reply:
Hmmm... then we have a problem. 

It was double encoding so we would need to double decode. 

So the user add it's email to Login, but when he reached AddDevice, the email was partially decoded, so he didn't automatically proceed, he had to fix his e-mail. 

It was only happening when testing the fast-auth-signer alone, if it was tested with near-discovery it didn't happen

Weird bug, I can revert it and we fix it later, I thought was a quick fix. But it not related to the PR